patches/common/chromium/can_create_window.patch

```diff
@@ -5,114 +5,29 @@ Subject: can_create_window.patch
 
 
 diff --git a/content/browser/frame_host/render_frame_host_impl.cc b/content/browser/frame_host/render_frame_host_impl.cc
-index 1a023385a3d27d9145f2a0a069ff6799c69ec66a..30328f5e62b4fdbb423a97454ee0591c076ac6bc 100644
+index 1a023385a3d27d9145f2a0a069ff6799c69ec66a..8289c09b9e3d30ae171942a987fbc1dca83f58d9 100644
 --- a/content/browser/frame_host/render_frame_host_impl.cc
 +++ b/content/browser/frame_host/render_frame_host_impl.cc
-@@ -3669,6 +3669,38 @@ void RenderFrameHostImpl::CreateNewWindow(
-                "frame_tree_node", frame_tree_node_->frame_tree_node_id(), "url",
-                params->target_url.possibly_invalid_spec());
- 
-+  scoped_refptr<network::ResourceRequestBody> body;
-+  if (params->body->has_object) {
-+    body = new network::ResourceRequestBody;
-+    std::vector<network::DataElement> elements;
-+    for (const auto& iter : params->body->elements) {
-+      network::DataElement element;
-+      switch (iter->type) {
-+        case network::DataElement::TYPE_BYTES: {
-+          element.SetToBytes(iter->bytes.data(), iter->bytes.length());
-+          break;
-+        }
-+        case network::DataElement::TYPE_FILE: {
-+          element.SetToFilePathRange(iter->path, iter->offset, iter->length,
-+                                     iter->expected_modification_time);
-+          break;
-+        }
-+        case network::DataElement::TYPE_BLOB: {
-+          element.SetToBlobRange(iter->blob_uuid, iter->offset, iter->length);
-+          break;
-+        }
-+        case network::DataElement::TYPE_DATA_PIPE:
-+        default:
-+          NOTREACHED();
-+          break;
-+      }
-+      elements.push_back(std::move(element));
-+    }
-+    body->swap_elements(&elements);
-+    body->set_identifier(params->body->identifier);
-+    body->set_contains_sensitive_info(params->body->contains_sensitive_info);
-+  }
-+
```

Comment:
can we really get rid of all this stuff? that's great!

Reply:
`network::ResourceRequestBody` got typemapped in https://chromium-review.googlesource.com/c/chromium/src/+/1395977 , which helped us get rid off these manual conversions. Side note: mojom typemaps are really cool feature :)